### ISSUE TITLE
🐞 Textfield is not behave properly

### ISSUE THREAD

**Issue by Strypper** [yiszza/Material.Components.Maui#24]
## What happened
I haven't had to fully text the `TextField` yet but this is what I'd pick up on Android Emulator. The input seems janky like in the video. The text won't show up until I paste the text from the clipboard and the `TextField` started to behave normally 

[Evidence](https://1drv.ms/v/s!Atp5h2QiqzrFid45u0QM6WH3u6eC5Q?e=Imz4Ol)

## Implemetation
```xml
                <mdc:TextField
                    IconKind="Pets"
                    IsError="{x:Binding PetForm.PetNameValid}"
                    LabelText="Cung cấp tên"
                    SupportingText="{x:Binding PetForm.PetNameInvalidMessage}"
                    Text="{x:Binding PetForm.PetName,
                                     Mode=TwoWay}" />
```

## Platform tested
1. Windows - fine 👍🏽
2. Android - issue happened 💀

**Comment by TimothyFran** [comment]
Have you managed to solve this? Having same problem.

[Screen recording](https://filedn.com/l75coKMbsVGJXJ84utNi3Gj/2023-07-02-11-26-32.mp4)

**Comment by Strypper** [comment]
> Have you managed to solve this? Having same problem.
> 
> [Screen recording](https://filedn.com/l75coKMbsVGJXJ84utNi3Gj/2023-07-02-11-26-32.mp4)

Not yet! hope the author that could look at this

**Comment by yiszza** [comment]
The current focus of work is #16 , where all controls are being refactored either simply or thoroughly. TextField is a big trouble, but I will try to make it work more perfectly when I turn my attention to it.

**Comment by TimothyFran** [comment]
> The current focus of work is #16 , where all controls are being refactored either simply or thoroughly. TextField is a big trouble, but I will try to make it work more perfectly when I turn my attention to it.

Hi, is there any workaround meanwhile?

**Comment by yiszza** [comment]
> > The current focus of work is #16 , where all controls are being refactored either simply or thoroughly. TextField is a big trouble, but I will try to make it work more perfectly when I turn my attention to it.
> 
> Hi, is there any workaround meanwhile?

Not now.

**Comment by yiszza** [comment]
Fixed in 158da7b

**Comment by Strypper** [comment]
> Fixed in 158da7b 

Cool how long will the nuget got updated ?

**Comment by yiszza** [comment]
One week? If I'm not busy.

**Comment by SamuelJoseBolanoMejia** [comment]
¿Ha conseguido solucionar esto?

**Comment by TimothyFran** [comment]
> Fixed in [158da7b](https://github.com/mdc-maui/mdc-maui/commit/158da7b140cfd05f63f09879aa09392d9133066f)

Hi, when will the nuget be updated?

**Comment by ehab-saad-neudesic** [comment]
Hi, any update on this issue

**Comment by DharunAR** [comment]
Hi Any update on this issue? I do face the same problem with the TextField on 0.1.2-beta version. May I know when this issue will be addressed?

**Comment by ehab-saad-neudesic** [comment]
Hi, any update here please?

**Comment by DharunAR** [comment]
@yiszza When will the next version of Nuget will be released with these fixes [158da7b](https://github.com/mdc-maui/mdc-maui/commit/158da7b140cfd05f63f09879aa09392d9133066f)?
